eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/GovSpec.hs

```diff
@@ -223,6 +225,78 @@ spec =
           enactState <- getsNES $ newEpochStateGovStateL . cgEnactStateL
           rsEnactState pulserRatifyState `shouldBe` enactState
 
+      xit "policy is respected by proposals" $ do
+        let
+          scriptHash = hashScript @era $ alwaysSucceeds @'PlutusV2 0
+          wrongScriptHash = hashScript @era $ alwaysSucceeds @'PlutusV2 1
+        modifyNES $
+          nesEsL . esLStateL . lsUTxOStateL . utxosGovStateL . cgEnactStateL . ensConstitutionL
+            .~ Constitution def (SJust scriptHash)
```

Comment:
We should not modify the state manually in imp state whenever we have a process for changing this part of the state by the means of submitting a transaction.

I'll address this when I'll be enabling the test, but I am just making a comment for the future work on imp tests.